### DISCUSSION TITLE
Create loop, repeat animation

### DISCUSSION THREAD

**Discussion by MilosGCM** [tomik23/circular-progress-bar#69] [Ideas]
Hello is it possible to create a loop? Repeat animation, for example, every 6 seconds... Or with IntersectionObserver repeat animation from the beginning every time when the section below the target shows(for example if the user scroll)?

**Comment by tomickigrzegorz** [comment]
@MilosGCM Have you looked at the examples and documentation? Probably not - [examples](https://tomik23.github.io/circular-progress-bar/) 😉
There you will find animations working on the scroll (IntersectionObserver) as well as on setInterval.

[intersectionobserver](https://github.com/tomik23/circular-progress-bar#intersectionobserver)
[update-progress-bar-dynamically](https://github.com/tomik23/circular-progress-bar#update-progress-bar-dynamically)
